### PR TITLE
auth

### PR DESCRIPTION
**Pull request by subharthihazra**

### CODE REVIEW

File: server/dist/errorhandlers/ErrorMiddleware.js.map
Comment (afeefuddin) [inline server/dist/errorhandlers/ErrorMiddleware.js.map:1]: What is this ??

File: server/dist/errorhandlers/CustomError.js.map
Comment (afeefuddin) [inline server/dist/errorhandlers/CustomError.js.map:1]: what is this ?

**Review by afeefuddin — COMMENTED** [review]
What are .js.map?

File: server/dist/errorhandlers/ErrorMiddleware.js.map
Comment (subharthihazra) [reply server/dist/errorhandlers/ErrorMiddleware.js.map:1]: this were some middlewere for error hadling in prev code .. i will merge or alter that after

**Comment by subharthihazra** [comment]
@afeefuddin look now

**Comment by afeefuddin** [comment]
Will you push the frontend part in the separate PR ? Have you completed the frontend of Auth?

**Pull request merged**